juniper_official/Solutions/EVPN-VXLAN/check-stp-state-netconf.rule

```diff
@@ -0,0 +1,108 @@
+/*
+ * Collects each member vlan STP state periodically and notifies anomalies when STP state
+ * is discarding.
+ *
+ * One input control detection
+ * 
+ *   1) input-vlan-id, is a regular expression that matches the vlan id that you would 
+ *      like to monitor.  By default it's '.*', which matches all interfaces. 
+ *      Use something like '100|107' to match only vlan IDs 100 and 107.
+ *
```

Comment:
s/control/controls/g
s/input-vlan-id,/input-vlan-id variable,/g

Reply:
updated the rule